### PR TITLE
Judith's intro blog post to the community - test build

### PR DESCRIPTION
**Pull request by judithpatudith**
I'm trying to test build my first blog post on the develop website! Let me do it please! OSS-94

### CODE REVIEW

**Comment by philipnrmn** [comment]
add to whitelist

File: src/blog/2016-09-07_community_manager.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+author: Judith Malnick, Mesosphere
+category: community
+description: Meet Mesosphere's new DC/OS Community Manager.
+layout: article.jade
+collection: posts
+lunr: true
+---
+# Hello, DC/OS Community!
+
+	Three weeks ago, on my first day as Mesosphere’s first official DC/OS Community Manager, I assumed that I was walking into a green-field community. DC/OS had only been open source for a few months, and many of the support structures that enable community action were not (and still are not) yet in place. I was expecting to start tackling questions like, “How do we attract members to our community?” and, “How can we motivate people to engage with DC/OS?”
```
Comment (philipnrmn) [inline src/blog/2016-09-07_community_manager.md]: Missing newline here (markdown requires double newlines for a paragraph break)

**Comment by philipnrmn** [comment]
I'm not sure whether it makes any difference on the website, but in github-flavoured markdown the leading tabs on your paragraphs are causing a rendering weirdness - see the [preview](https://github.com/judithpatudith/dcos-website/blob/5c572d6ee9b516ac61f83266707c0cd9e4700394/src/blog/2016-09-07_community_manager.md). Removing them fixes it.

**Comment by judithpatudith** [comment]
@philipnrmn I think this is done, do you mind reviewing and (hopefully) approving it? Thank you for all your help!

File: src/blog/2016-09-07_community_manager.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+On my first day as Mesosphere’s first official DC/OS Community Manager, I assumed that I was walking into a green-field community. DC/OS had only been open source for a few months, and many of the support structures that enable community action were not (and still are not) yet in place. I was expecting to start tackling questions like, “How do we attract members to our community?” and, “How can we motivate people to engage with DC/OS?”
+
+As everyone who is reading this knows, these questions were pretty naive. The DC/OS community is up and running: the community slack has 2,680 members, the mailing list has 290 members, and DC/OS had over 60 launch partners for its open-source debut on April 19th. Mesosphere is one stakeholder of thousands in the DC/OS community, and all of those stakeholders are as passionate about using and contributing to DC/OS as we are. My questions have changed in the last few weeks. Now, I’m also asking myself what I can do to help the DC/OS community succeed, and what I can do to help Mesosphere be a good member, and steward of that community. I’d like to propose three things I can do in the short term, and ask for your help and advice.
```
Comment (philipnrmn) [inline src/blog/2016-09-07_community_manager.md]: Extraneous comma after 'be a good member'.

**Comment by philipnrmn** [comment]
One tiny nit, then :shipit:

**Comment by judithpatudith** [comment]
Done!

**Comment by philipnrmn** [comment]
Just waiting on the build :)

**Comment by philipnrmn** [comment]
https://dev.dcos.io/blog/2016/hello-dc-os-community/index.html ✨

**Pull request merged**